layouts/partials/mithology/mithology.html

```diff
@@ -7,6 +7,7 @@ <h2 class="h4 h4-primary u-mb-32">Una mica de mitologia</h2>
         propis vicis, virtuts i capacitats.
       </p>
       <p class="h6 h6-bold u-mb-32">Som com érem i érem com som.</p>
+      <p class="h6 h6-bold u-mb-32" style="font-size: small">"La cabra siempre tira al monte. —Anónimo"</p>
```

Comment:
Avoid using inline styles! (again, we don't want to play the "specificity wars")